resources/views/dashboard/admin/live.blade.php

```diff
@@ -26,7 +26,7 @@
         <?php
             $pub = ($liveInfo->publish) ? 'checked' : '';
         ?>
-        <label class="switch">
+        <label class="switch mb-3">
```

Comment:
See above

Reply:
"mb-3" applied to a switch only occurs in 1/3 instances that this shows up in our codebase

Reply:
Done